src/routes/+page.svelte

```diff
@@ -67,7 +67,7 @@
 			<FeatureCard title="Welcome to swyxkit 2022!" href="/welcome" stringData="Jan 2022" />
 			<FeatureCard
 				title="Moving to a GitHub CMS"
-				href="/moving-to-a-git-hub-cms"
+				href="/moving-to-a-github-cms"
```

Comment:
difference between sindresorhus/slugfiy and this new slugify

technically a breaking change...

Reply:
Correct, the way it handles the camel case is better I guess.